praktikumsplaner-backend/src/test/java/de/muenchen/oss/praktikumsplaner/validators/StartDateBeforeEndDateValidatorTest.java

```diff
@@ -0,0 +1,103 @@
+package de.muenchen.oss.praktikumsplaner.validators;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import de.muenchen.oss.praktikumsplaner.domain.Meldezeitraum;
+import de.muenchen.oss.praktikumsplaner.dtos.CreateMeldezeitraumDTO;
+import de.muenchen.oss.praktikumsplaner.dtos.MeldezeitraumDTO;
+import jakarta.validation.Validation;
+import jakarta.validation.Validator;
+import jakarta.validation.ValidatorFactory;
+import java.time.LocalDate;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class StartDateBeforeEndDateValidatorTest {
```

Comment:
these tests doenst test the `StartDateBeforeEndDateValidator` directly. For the tests maintenance is required to handle the other rules, name for example, too.

Pls refactor the test that only the Validator is the unit under test.